docs/version0.32.md

```diff
@@ -53,6 +54,10 @@ This option specifies whether generated hidden MethodHandle frames are displayed
 
 You can now use the `SharedClassStatistics` API to get the name, path, and directory of a shared classes cache. Depending on the operating system, you can also get the number of attached VMs for a non-persistent cache. This information is available through the following new methods: `cacheDir()`, `cacheName()`, `cachePath()`, and `numberAttached()`. For more information, see the API documentation.
 
+### New `-XX:[+|-]OpenJ9CommandLineEnv` option added
+
+This option specifies whether the command line should be captured in the environment variable `OPENJ9_JAVA_COMMAND_LINE`. For more information, see [`-XX:[+|-]OpenJ9CommandLineEnv`](xxopenj9commandlineenv.md).
```

Comment:
Can we please change "should be" to "is" (use present tense where possible), or even better, use similar text to xxopenj9commandlineenv.md, ie "VM captures the command line" rather than "command line is captured" (active voice). Also, do we mean the generic "command line" or more specifically the "command that was used to start the VM"?
"

Reply:
I will update as suggested.

Yes, it's the more generic command line. It applies not just to the `java` launcher, but to any application that uses JNI to invoke `JNI_CreateJavaVM()`.